### PR TITLE
Show better errors when [data-page] does not exist

### LINKED ISSUE #11: Show better errors when `[data-page]` does not exist
In my CI test. It was hard to figure out that one of my layouts' `<body>` tag was missing the `[data-page]` attribute. The error showed something like:

```
Uncaught TypeError: Cannot read property 'split' of null
```

I suggest something like:

```
Uncaught Error: <body> does not have a [data-page] attribute
```

What do you think? I have `page.js` version that uses the one mentioned above. Let me know if you want this and I can submit a PR.

Thanks.

### PR DESCRIPTION
**Pull request by westoque**
Closes #11

### CODE REVIEW

File: src/page.js [hunk 1]
```diff
@@ -61,13 +61,14 @@ export class Page {
   // Returns nothing.
   dispatch(data = {}) {
     const raw = this.recognize();
+    if (raw === void(0) || raw === null) {
+      throw new Error('<body> does not have a [data-page] attribute');
```
Comment (lucasmazza) [inline src/page.js]: can we move this to an `else` clause and stick with a single conditional?

Comment (westoque) [inline src/page.js]: @lucasmazza sure thing. although on "Chrome 51.0.2704.103 (64-bit)", when `[data-page]` attribute does not exist on `<body>`, `raw` would evaluate to `null`. So to stick with a single conditional, we can instead do maybe `if (!raw) { ... }`, would you prefer this instead? thanks.

Comment (lucasmazza) [inline src/page.js]: We can go with `if (raw)` and leave the throw on the else section.

Comment (westoque) [inline src/page.js]: @lucasmazza ah. thanks for the fast response. 

btw, the ci is failing and getting the same errors as i am locally. will try to look at that too.

Comment (lucasmazza) [inline src/page.js]: Master should be green by now - I pushed some fixes earlier. You can rebase your branch and everything should be fine.

Comment (westoque) [inline src/page.js]: @lucasmazza Just did that, and pushed the changes. Thanks.

**Comment by lucasmazza** [comment]
Thanks! :)

**Pull request merged**